bin/github-review.sh

```diff
@@ -0,0 +1,313 @@
+fi
+if ! command -v gh &> /dev/null; then
+    echo "ERROR: Please install the 'gh' GitHub CLI" 1>&2
+    exit 1
+fi
+if ! command -v cast &> /dev/null; then
+    echo "ERROR: Please install the 'cast' Foundry (forge)" 1>&2
+    exit 1
+fi
+
+pr=0
+chainid=0
+rpc=0
+version=""
+# TODO: "1.0.0" "1.1.1" "1.2.0"
+availableVersion=("1.3.0" "1.4.1")
+case $# in
+    4)
+        if ! [[ $1 =~ ^[0-9]+$ ]]; then
+            echo "ERROR: $1 is not a valid GitHub PR number" 1>&2
+            usage
+            exit 1
+        fi
+        pr=$1
+        if ! [[ $2 =~ ^[0-9]+$ ]]; then
+            echo "ERROR: $2 is not a valid Chain ID number" 1>&2
+            usage
+            exit 1
+        fi
+        chainid="$(cast chain-id --rpc-url $3)"
+        if [[ $chainid != $2 ]]; then
+            echo "ERROR: RPC $3 doesn't match chain ID $2" 1>&2
+            usage
+            exit 1
+        fi
+        rpc=$3
+        if [[ ! " ${availableVersion[@]} " =~ " ${4} " ]]; then
+            echo "ERROR: $4 is not a valid version" 1>&2
+            usage
+            exit 1
+        fi
+        version=$4
+        ;;
+    *)
+        usage
+        exit 1
+        ;;
+esac
+
+echo "Verifying Deployment Asset"
+gh pr diff $pr --patch | git apply --include 'src/assets/**'
+
+# Getting default addresses
+
+# v1.3.0 & v1.4.1
+cfh=$(jq -r .defaultAddress src/assets/v$version/compatibility_fallback_handler.json)
+cc=$(jq -r .defaultAddress src/assets/v$version/create_call.json)
+msco=$(jq -r .defaultAddress src/assets/v$version/multi_send_call_only.json)
+ms=$(jq -r .defaultAddress src/assets/v$version/multi_send.json)
+sml=$(jq -r .defaultAddress src/assets/v$version/sign_message_lib.json)
+sta=$(jq -r .defaultAddress src/assets/v$version/simulate_tx_accessor.json)
+
+# v1.3.0
+gsl2=0
+gs=0
+pf=0
+if [[ $version == "1.3.0" ]]; then
+    gsl2=$(jq -r .defaultAddress src/assets/v$version/gnosis_safe_l2.json)
+    gs=$(jq -r .defaultAddress src/assets/v$version/gnosis_safe.json)
+    pf=$(jq -r .defaultAddress src/assets/v$version/proxy_factory.json)
+fi
+
+# v1.4.1
+sl2=0
+spf=0
+s=0
+if [[ $version == "1.4.1" ]]; then
+    sl2=$(jq -r .defaultAddress src/assets/v$version/safe_l2.json)
+    spf=$(jq -r .defaultAddress src/assets/v$version/safe_proxy_factory.json)
+    s=$(jq -r .defaultAddress src/assets/v$version/safe.json)
+fi
+
+# Getting default address based on the chain id
+
+# v1.3.0 & v1.4.1
+cfhCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/compatibility_fallback_handler.json)
+ccCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/create_call.json)
+mscoCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/multi_send_call_only.json)
+msCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/multi_send.json)
+smlCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/sign_message_lib.json)
+staCI=$(jq -r '.networkAddresses["'$chainid'"]' src/assets/v$version/simulate_tx_accessor.json)
```

Comment:
I feel like this can be simplified to a loop over the JSON files, and have the equality check be done in `jq`.